build.gradle.kts

```diff
@@ -69,3 +69,7 @@ kotlin {
     }
   }
 }
+
+tasks.named<Test>("jvmTest") {
```

Comment:
Tests couldn't execute so I added this. Idk, if this or the invalidate & restart fixed the issues